src/Controls/Legend.js

```diff
@@ -3,8 +3,8 @@ EsriLeaflet.LegendControl = L.Control.extend({
   options: {
     listTemplate: '<ul>{layers}</ul>',
     layerTemplate: '<li><strong>{layerName}</strong><ul>{legends}</ul></li>',
-    listRowTemplate: '<li><img width="{width}" height="{height}" src="data:{contentType};base64,{imageData}"><span>{label}</span></li>',
-    emptyLabel: '<all values>',
+    listRowTemplate: '<li><img alt="map legend icon representing {label}" width="{width}" height="{height}" src="data:{contentType};base64,{imageData}"><span>{label}</span></li>',
```

Comment:
I don't like hard-coded English phrases in the template, it increases the accessibility, but affects the localisation, is there a way around it?

Reply:
I agree with this.  Would adding an option that let's the user specify the text of the `alt` attribute be a reasonable solution?

Something like the following:
```js
L.esri.Controls.legend([dynamicLayer, featureLayer], {
   position: 'topright',
   altTextTemplate: 'icon representing a map layer'
}).addTo(map);
```

I would still want to use the `label` property, but understand it could get tricky without knowing what the user would pass in?  

Any thoughts or suggestions?

Reply:
it's a good suggestion! But probably it would require a second pass to substitute? or it can be replaced where it's added.
```js
    listRowTemplate: '<li><img alt="'+ altTextTemplate +'" width="{width}" height="{height}" src="data:{contentType};base64,{imageData}"><span>{label}</span></li>',
```